frontend/pages/DashboardPage/cards/Munki/Munki.tsx

```diff
@@ -109,8 +109,8 @@ const Munki = ({
                         <>
                           To see Munki versions, deploy&nbsp;
                           <CustomLink
-                            url="https://fleetdm.com/docs/using-fleet/adding-hosts#osquery-installer"
-                            text="Fleet's osquery installer"
+                            url="https://fleetdm.com/learn-more-about/fleetd"
+                            text="Fleet's agent (fleetd)"
```

Comment:
Similar comment as the MDM versions comment:

The tables used to get Munki info is only included in fleetd.

What could we update this empty state to say instead?